tests/nnx/module_test.py

```diff
@@ -91,8 +91,8 @@ class Foo(nnx.Pytree):
   def test_consistent_attrs_dataclass(self):
     @dataclasses.dataclass
     class Foo(nnx.Pytree):
-      a: nnx.Data[int]
```

Comment:
Will `nnx.Data` and `nnx.Static` continue to work until we remove them?  If so, should we leave in tests of `nnx.Data` and `nnx.Static` until then?

Reply:
Flax CI raises on warnings so we are forced to fix the test. I'll try to add a warning ignore and duplicate the test.